### DISCUSSION TITLE
Need OrthographicCamera in ui.scene()

### DISCUSSION THREAD

**Discussion by vitamingcheng** [zauberzeug/nicegui#2872] [Ideas / Feature Requests]
I want to show some engineering models in scene, these models usually use orthogonal view, is there any plan to support orthogonal camera in ui.scene()? 

thanks a lot!

**Comment by falkoschindler** [comment]
Hi @vitamingcheng,

We haven't thought about orthographic cameras in `ui.scene`, but it would certainly be a great extension. I guess we could introduce some initializer parameter like `camera_type: Literal['perspective', 'orthographic'] = 'perspective'` for `ui.scene`.

While thinking about it, I decided to quickly try it out: See PR https://github.com/zauberzeug/nicegui/pull/2890. What do you think?

**Reply by vitamingcheng** [reply]
Sorry for not replying in time, I checked out the revision file submitted in PR [#2890](https://github.com/zauberzeug/nicegui/pull/2890) and did a simple test. Here's a comparison of the two camera views, it is what I expected, thank you very much for your help!

![图片1](https://github.com/zauberzeug/nicegui/assets/36910447/fe5b3d89-6e3d-47c5-9c2d-d409e62e6ae3)

**Reply by falkoschindler** [reply]
Great! PR #2890 will be part of the next release. Therefore I'll close this feature request as completed. ✔️
